Makefile

```diff
@@ -6,13 +6,14 @@ ADAPTER_REST_PATH=cmd/adapter-rest
 
 # Namespace for the agent images
 DOCKER_OUTPUT_NS   ?= docker.pkg.github.com
+ADAPTER_REST_BASE_IMAGE_NAME ?= trustbloc/edge-adapter/adapter-rest-base
 ISSUER_ADAPTER_REST_IMAGE_NAME   ?= trustbloc/edge-adapter/issuer-adapter-rest
 RP_ADAPTER_REST_IMAGE_NAME   ?= trustbloc/edge-adapter/rp-adapter-rest
 MOCK_ISSUER_IMAGE_NAME ?= mock-issuer
 
 # Tool commands (overridable)
-ALPINE_VER ?= 3.11
-GO_VER ?= 1.14
+ALPINE_VER ?= 3.12
+GO_VER ?= 1.14.6
```

Comment:
The tag 1.14 will point to the latest minor version 1.14.x. (currently it's [1.14.6](https://github.com/docker-library/golang/blob/3dc1f97e6a772b44b813edd780e53af8a0df799f/1.14/alpine3.12/Dockerfile#L11)).

Reply:
@rolsonquadras done